modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -7,6 +7,6 @@ In API Designer, you can use two different tools to create API specifications in
 
 API editor:: Use this tool if you do have experience with writing API specifications. It offers a larger degree of freedom for creating more complex specifications. It also displays errors and warnings as problems in your specification occur.  The steps for using this tool are described in xref:design-create-publish-api-raml-editor.adoc[Create and Publish an API Specification with the API Editor in API Designer].
 
-Visual API editor:: Use this tool if you do not have much experience with writing API specifications. As you fill in various forms, the editor writes your API specification for you. The steps for using this tool are described in xref:design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual API Editor in API Designer].
+Visual API editor:: Use this tool for lightweight designing and creating scaffolds. This editor does not allow you to import RAML fragments, add security schemes, or perform several of the tasks that the API editor allows. As you fill in various forms, the editor writes your API specification for you. After creating a scaffold in this editor, you can switch to the API editor to customize your specification. The steps for using this tool are described in xref:design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual API Editor in API Designer].
```

Comment:
I would remove the "add security schemes" limitation as, in fact, we will be adding this in the next version

Reply:
Done.